use/PopDiffSequenceData.Rmd

```diff
@@ -113,6 +119,10 @@ With these exploratory analyses, we understand how genetic diversity is distribu
 - Margarita M. López- Uribe (Author)
 - Zhian N. Kamvar (edits)
 
+# References
+
+Excoffier L., Smouse PE., Quattro JM. 1992. Analysis of molecular variance inferred from metric distances among dNA haplotypes: Application to human mitochondrial dNA restriction data. Genetics 131:479–491. Available at: http://www.genetics.org/content/131/2/479.abstract
```

Comment:
Yes, but I couldn't find a DOI for this particular paper. Genetics seems to be a rebel that doesn't play by the rules. Instead of a DOI, they use a permalink.